spec/models/conference_room_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe ConferenceRoom, type: :model do
+  describe "validation" do
+
```

Comment:
This file has poor formatting:
1. multiple newlines
2. lack of newlines  (eg. lines 29-30)
3. different formatting conventions in `describe` blocks